### ISSUE TITLE
Loss function from logs

### ISSUE THREAD

**Issue by dionman** [vanderschaarlab/synthcity#195]
Is there a unified, maximally model-agnostic way to recover the training/validation loss function(s) values on logging iterations of a trained model upon saving?

**Comment by robsdavis** [comment]
Unfortunately not at the moment. The loss is sent to the log at the debug level for most models, but there is no infrastructure for retrieving these values.
